app/webpacker/controllers/products_controller.js

```diff
@@ -35,8 +45,15 @@ export default class extends ApplicationController {
   };
 
   getLoadingController = () => {
+    let loadingSpinner = null;
+    try {
+      loadingSpinner = this.loadingTarget; // throws missing loading target error
```

Comment:
- When I confirm the deletion via the modal, for some reason, the `loadingTarget` is not getting set and hence the loader doesn't show up.
- I've validated that the HTML document does have the `loading`  data-target value when the delete modal is being displayed, but not sure why it's not getting set in the controller for the delete confirmation
- So I came up with this approach
- I'm new to Stimulus and Reflex so that's why I might be missing something here.😅
- Let me explore some more if I can avoid this.

Reply:
Fair enough, I'm still getting used to it too! Thanks for looking into it.

Reply:
It's been updated https://github.com/openfoodfoundation/openfoodnetwork/pull/11846/commits/ec5df1a0d8584f5282e8a8f5175c05dab123069a